sdk/core/src/main/java/com/klaviyo/coresdk/networking/requests/NetworkRequest.kt

```diff
@@ -67,17 +77,19 @@ internal abstract class NetworkRequest {
                 val outputStream = connection.outputStream
                 val writer = BufferedWriter(OutputStreamWriter(outputStream, "UTF-8"))
                 writer.use {
-                    it.write(payload)
+                    it.write(payload!!)
```

Comment:
Why is this necessary?

Reply:
Suppresses a nullable warning. The if statement above should not allow this to ever be null or empty here